ansible/roles/haproxy/tasks/manifests.yml

```diff
@@ -1,21 +1,21 @@
 ---
-- name: 'haproxy | ensure kubernetes manifests directory "{{ cluster_kubernetes_manifests_path }}" exists'
+- name: 'manifests | ensure kubernetes manifests directory "{{ cluster_kubernetes_manifests_path }}" exists'
   ansible.builtin.file:
     path: '{{ cluster_kubernetes_manifests_path }}'
```

Comment:
`cluster_kubernetes_manifests_path` isn't being initialized in defaults or the group vars